.changeset/loud-bugs-invite.md

```diff
@@ -0,0 +1,5 @@
+---
+"@pnpm/plugin-commands-installation": patch
+---
+
+Remove workspaceDir args from update handler in the test because update command works as recursive with both of workspaceDir arg and dir arg"
```

Comment:
there is no need in a changeset file when you only change tests.

Reply:
Sorry, I did not know that. removed in [3964007](https://github.com/pnpm/pnpm/pull/5457/commits/3964007ba25d3b634a225222a255b838627673e7) 👍